raml-parser-2/src/main/java/org/raml/v2/internal/impl/v10/phase/ExampleValidationPhase.java

```diff
@@ -178,6 +186,26 @@ private boolean isJsonValue(String value)
         return value.trim().startsWith("{") || value.trim().startsWith("[");
     }
 
+    private boolean isValidJson(String value)
+    {
+        return getJsonError(value) == null;
+    }
+
+    @Nullable
+    private String getJsonError(String value)
+    {
+        try
+        {
+            final ObjectMapper mapper = new ObjectMapper();
+            mapper.readTree(value);
```

Comment:
The problem with this approach is that we end up parsing twice the value. In case of the parser (example validation) is not a big deal. The problem is going to be API kit that validates incoming payloads. We need to think how we can avoid this.